### PR TITLE
Add missing Xcode projects to samples

### PR DESCRIPTION
**Pull request by MSFTFox**
Add missing xcode projects to GLKit samples.

---
This change is [Reviewable](https://reviewable.io/reviews/microsoft/winobjc/1659)

### CODE REVIEW

**Comment by rajsesh-msft** [comment]
samples/GLKitComplex/GLKitComplex/Data/lobbyxneg.JPG, etc, what exactly changed and why?

File: samples/HelloOpenGL/HelloOpenGL.vsimporter/HelloOpenGL-WinStore10/HelloOpenGL.vcxproj [hunk 1]
```diff
@@ -29,74 +28,78 @@
     <EnableDotNetNativeCompatibleProfile>true</EnableDotNetNativeCompatibleProfile>
     <WindowsTargetPlatformVersion>10.0.10586.0</WindowsTargetPlatformVersion>
     <WindowsTargetPlatformMinVersion>10.0.10586.0</WindowsTargetPlatformMinVersion>
-    <ProjectGuid>{94BA35CD-F4E9-4B5B-BEE7-A452AB514360}</ProjectGuid>
-    <WINOBJC_SDK_ROOT>..\..\..\..</WINOBJC_SDK_ROOT>
+    <Keyword>IslandwoodProj</Keyword>
+    <ProjectGuid>{5F8B73C6-1253-4481-9567-A81321309725}</ProjectGuid>
+    <RootNamespace>HelloOpenGL</RootNamespace>
+    <WINOBJC_SDK_ROOT Condition="'$(WINOBJC_SDK_ROOT)' == ''">E:\island\github\henryfox\WinObjC</WINOBJC_SDK_ROOT>
```
Comment (rajsesh-msft) [inline samples/HelloOpenGL/HelloOpenGL.vsimporter/HelloOpenGL-WinStore10/HelloOpenGL.vcxproj]: >E:\island\github\henryfox\WinObjC [](start = 62, length = 33)

Please be sure to import using -relativepath, or replace with ..\..\..\...  Here and elsewhere.

**Comment by rajsesh-msft** [comment]
:clock1:

**Comment by MSFTFox** [comment]
Nothing changed to the actual files but git refuses to acknowledge that they're still the same... I don't know why.

---
In reply to: [271397354](https://github.com/Microsoft/WinObjC/pull/1659#issuecomment-271397354) [](ancestors = 271397354)

**Pull request merged**